lib/ansible/modules/cloud/ovirt/ovirt_vm.py

```diff
@@ -1076,8 +1082,11 @@ def build_entity(self):
                         otypes.BootDevice(dev) for dev in self.param('boot_devices')
                     ],
                 ) if self.param('boot_devices') else None,
+                cmdline=self.param('kernel_params'),
```

Comment:
I think the ` if self.param('kernel_params_persist') ` should be also here for cmdline

Reply:
It was there but after one of your comments I removed it 

I'll correct it.